ios/RNSScreenStack.m

```diff
@@ -520,10 +520,8 @@ - (void)dismissOnReload
     screen = (RNSScreenView *)fromVC.view;
   }
   if (screen != nil &&
-      (screen.stackAnimation == RNSScreenStackAnimationFade ||
-       screen.stackAnimation == RNSScreenStackAnimationSimplePush ||
-       screen.stackAnimation == RNSScreenStackAnimationNone ||
-       screen.stackAnimation == RNSScreenStackAnimationSlideFromBottom)) {
+      (screen.stackAnimation != RNSScreenStackAnimationFlip && screen.stackAnimation != RNSScreenStackAnimationNone &&
```

Comment:
why is flip animation excluded here?

Reply:
I changed it to exclude the animations handled by the system in order not to add new items to `if` clause when adding new animations. Also, `RNSScreenStackAnimationNone` should not be in here, will change it.